docs/manage/configuration.md

```diff
@@ -298,9 +298,12 @@ In the `Raw JSON` configuration, you will see the fragment ID in the `fragments`
 
 For an example of adding a fragment to a robot, see the [Viam Rover fragment tutorial](/try-viam/rover-resources/rover-tutorial-fragments/).
 
-<!--
 ## Auth/network
--->
+
+The auth section allows you to configure robot-level API keys for connecting over LAN.
+
+The network section allows you to set the address `viam-server` binds to for accepting connections.
+By default, `viam-server` binds to `0.0.0.0:8080`.

```

Comment:
can we add "when managed by app.viam.com or when authentication and TLS are enabled"? Otherwise this may look insecure by default.